sdk/core/perf/test/src/random_stream_test.cpp

```diff
@@ -35,7 +35,8 @@ TEST(circular_stream, basic)
 
   // 4nd read
   count = r_stream->Read(buffer, chunk, Azure::Core::Context::ApplicationContext);
-  EXPECT_EQ(count, 0);
```

Comment:
I think `EXPECT_EQ(count 0)` should more readable in failure logs.

Is it the warning about signed/unsigned mismatch why you are introducing this change? I've seen some of these warnings from gtest recently where I don't remember seeing them. It must be that they've broke something, and we are getting that updated version voa fetchcontent.

If that signed/unsigned comparison mismatch is the only case why you are changing this, try changing it to `EXPECT_EQ(count, 0U)` - should be more readable.

Otherwise, make expected `const`.

Needless to say, if this is an accidental change, `EXPECT_EQ(count, 0)` was the best of them all.

Reply:
yes, is the warning from msvc because comparing size_t to int.  Will try adding U